tests/func/clean/security_group_rule.yml

```diff
@@ -0,0 +1,8 @@
+- name: remove osm_security_group_rule
+  os_security_group_rule:
+    cloud: "{{ item }}"
+    security_group: default
```

Comment:
We should leave the default group unchanged and instead use our osm_security_group here.

Reply:
done